apps/console/src/features/users/components/add-user.tsx

```diff
@@ -324,22 +246,33 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
                                 showPassword={ t("common:showPassword") }
                                 type="password"
                                 value={ isPasswordGenerated ? randomPassword : initialValues?.newPassword }
-                                listen={ handlePasswordChange }
-                                loading={ isPasswordRegExLoading }
-                                validation={ (value: string, validation: Validation) => {
-                                    if (!isPasswordPatternValid) {
+                                validation={ async (value: string, validation: Validation) => {
+                                    let passwordRegex = "";
+                                    if (userStore !== "primary") {
```

Comment:
There's a constant for this in user store constants in the core module. Let's use that here.

Reply:
Resolved by ac16628